cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagementSettings.scala

```diff
@@ -3,10 +3,18 @@
  */
 package akka.cluster.http.management
 
+import java.net.InetAddress
+
 import com.typesafe.config.Config
 
 final class ClusterHttpManagementSettings(val config: Config) {
   private val cc = config.getConfig("akka.cluster.http.management")
   val ClusterHttpManagementPort = cc.getInt("port")
-  val ClusterHttpManagementHostname = cc.getString("hostname")
+  val ClusterHttpManagementHostname = {
+    val hostname = cc.getString("hostname")
+    if (hostname == "")
```

Comment:
Great. Can you also amend the comment section in reference.conf and mention that empty string in hostname has a special meaning?